toponetx/algorithms/spectrum.py

```diff
@@ -262,7 +263,29 @@ def simplicial_complex_hodge_laplacian_spectrum(
     return laplacian_spectrum(SC.hodge_laplacian_matrix(rank=rank))
 
 
-def cell_complex_adjacency_spectrum(CC: CellComplex, rank):
+def path_complex_hodge_laplacian_spectrum(
+    PC: PathComplex, rank: int, weight: str | None = "weight"
+):
+    """Return eigenvalues of the Laplacian of PC.
+
+    Parameters
+    ----------
+    PC : PathComplex
+    rank : int
+    weight : str or None, default='weight'
+        If None, then each cell has weight 1.
+
+    Returns
+    -------
+    np.ndarray
+        Eigenvalues.
+    """
+    return laplacian_spectrum(
+        PC.hodge_laplacian_matrix(rank=rank, signed=True), weight=weight
+    )
+
+
+def cell_complex_adjacency_spectrum(CCC: CellComplex, rank):
```

Comment:
- Missing type for `rank`.
- Cell complexes are abbreviated `CC`, not `CCC`.

```suggestion
def cell_complex_adjacency_spectrum(CC: CellComplex, rank: int):
```